weblate/addons/tests.py

```diff
@@ -1059,7 +1082,7 @@ def test_language_consistency(self) -> None:
         # Add one language
         language = Language.objects.get(code="af")
         self.component.add_new_language(language, None)
-        # To check: Failing on BulkForm
+        # To check #
```

Comment:
Probably, comment can be removed now?

Reply:
This is the only test that is failing, I looked into it and when we add one language `af`

The `<Translation: Test/TS — Afrikaans>` is missing from `Translation.objects` that is causing it to fail.

Reply:
That means that the add-on is not properly executed as it should be. It didn't create the additional translations.

Reply:
I currently don't see an error, but the problem can be in following paths:

* The project scoped add-on is not correctly installed within project.
* The project scoped add-ons are not picked up when filtering for execution.
* The language consistency add-on execution is broken.

Reply:
Yes, Thanks for the comment. I am looking into it.